content/docs/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -36,10 +36,61 @@ Pack contents are periodically synchronized with Palette.
 Please ensure that the ports 443 and 80 are exclusively allocated to the registry server and are not in use by other processes.
 </WarningBox>
 
-# Deploying a Pack Registry Server
-
-Palette provides a Docker image for the pack registry server. The following steps need to be performed to deploy the pack registry server using this docker image:
-
+# Deploying a Pack Registry Server with letsencrypt certificates
+
+The following section demonstrates how you can deploy a registry server secured with a TLS certificate issued by [Let's Encrypt](https://letsencrypt.org/).
+For more advanced configuration and deployment with self-signed certificates see [advanced configuration.](/registries-and-packs/advanced-configuration)
+Palette provides a Docker image for the pack registry server.
+The following steps need to be performed to deploy the pack registry server using this docker image:
+
+
+1. Create an httppasswd file 
+
+```
+mkdir spectropaxconfig
```

Comment:
This step reads better if split it out. Here is a previous example. 

![CleanShot 2023-03-29 at 16 21 49](https://user-images.githubusercontent.com/29551334/228689415-bfcf9f68-d26e-4e86-b126-20a1ac3abe2b.png)